### ISSUE TITLE
Is it possible to merge two dataframes?

### ISSUE THREAD

**Issue by gautambak** [vaexio/vaex#1296]
Hi there,

In pandas I'm joining two datrafarmes using merge, basically this:
`new_df = pd.merge(df, df2,  how='left', on="ticker")`

but it keeps running out of memory so I thought I'd try vaex for the same item but I'm unsure how to merge. I searched the documentation and couldn't find anything. I looked around the issues and it looks like people can merge but it's not being explained how.  

Is it possible and if so, can you direct me to a resource so can learn how to do it?

**Comment by maartenbreddels** [comment]
Hi,

yes, it's calling joining in Vaex: https://vaex.io/docs/tutorial.html#Joining

If anything unclear, let us know!

Regards,

Maarten

**Comment by gautambak** [comment]
Thank you this is helpful. I'm trying to send it a list for the on parameter but it's saying it only wants a string:
`df1.join(df2, how='left', on=groupsForAgg)`

Is there a work around for this? I have a process that cycles through a dataframe with different groupings

**Comment by JovanVeljanoski** [comment]
What are you trying to do ? What does the list contain?
Join is used for joining two dataframes, on a particular key (or column). 

If by sending a list you meant to do join on multiple keys, this is unfortunately not yet supported (looking for funding). 

If you had an entirely different use-case in mind, can you please explain it a bit more?

**Comment by gautambak** [comment]
Yes it was to do a join on multiple keys so the list contained multiple column names.
